### DISCUSSION TITLE
Terminal Error - `false` unknown or invalid utility

### DISCUSSION THREAD

**Discussion by EmpireNetworks** [jaredh159/tailwind-react-native-classnames#208] [Q&A] [answered]
I am experiencing an error when using twrnc on a React Native/Expo project. I get the below errors whenever the package is included in my project.  When I remove twrnc from my project and use standard css styles, the error is no longer there. I have looked everywhere for this issue but have not found a solution. 

ERROR MESSAGE IN VSCODE TERMINAL

`false` unknown or invalid utility
at node_modules/twrnc/dist/cjs/helpers.js:190:16 in consoleWarn
at node_modules/twrnc/dist/cjs/ClassParser.js:332:26 in parseUtility
at node_modules/twrnc/dist/cjs/ClassParser.js:111:36 in parse
at node_modules/twrnc/dist/cjs/create.js:74:34 in style
at components/NavOptions.js:40:25 in FlatList.props.renderItem
at node_modules/react-native/Libraries/Lists/FlatList.js:607:25 in renderer
at node_modules/react-native/Libraries/Lists/FlatList.js:640:25 in virtualizedListRenderKey
at node_modules/react-native/Libraries/Lists/VirtualizedList.js:2068:23 in CellRenderer#_renderElement
at node_modules/react-native/Libraries/Lists/VirtualizedList.js:2095:39 in CellRenderer#render
at node_modules/react-redux/lib/utils/Subscription.js:19:11 in notify
at node_modules/react-redux/lib/utils/Subscription.js:92:20 in notifyNestedSubs
at node_modules/react-redux/lib/utils/Subscription.js:97:32 in handleChangeWrapper
at node_modules/redux/lib/redux.js:305:14 in dispatch
at node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js:451:25 in <anonymous>
at node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js:390:39 in <anonymous>
at screens/HomeScreen.js:37:20 in GooglePlacesAutocomplete.props.onPress
at node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js:270:25 in request.onreadystatechange
at node_modules/react-native/Libraries/Network/XMLHttpRequest.js:637:22 in setReadyState
at node_modules/react-native/Libraries/Network/XMLHttpRequest.js:395:24 in __didCompleteResponse
at node_modules/react-native/Libraries/vendor/emitter/EventEmitter.js:105:35 in emit

MY PACKAGE.JSON FILE 

`false` unknown or invalid utility
at node_modules/twrnc/dist/cjs/helpers.js:190:16 in consoleWarn
at node_modules/twrnc/dist/cjs/ClassParser.js:332:26 in parseUtility
at node_modules/twrnc/dist/cjs/ClassParser.js:111:36 in parse
at node_modules/twrnc/dist/cjs/create.js:74:34 in style
at components/NavOptions.js:40:25 in FlatList.props.renderItem
at node_modules/react-native/Libraries/Lists/FlatList.js:607:25 in renderer
at node_modules/react-native/Libraries/Lists/FlatList.js:640:25 in virtualizedListRenderKey
at node_modules/react-native/Libraries/Lists/VirtualizedList.js:2068:23 in CellRenderer#_renderElement
at node_modules/react-native/Libraries/Lists/VirtualizedList.js:2095:39 in CellRenderer#render
at node_modules/react-redux/lib/utils/Subscription.js:19:11 in notify
at node_modules/react-redux/lib/utils/Subscription.js:92:20 in notifyNestedSubs
at node_modules/react-redux/lib/utils/Subscription.js:97:32 in handleChangeWrapper
at node_modules/redux/lib/redux.js:305:14 in dispatch
at node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js:451:25 in <anonymous>
at node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js:390:39 in <anonymous>
at screens/HomeScreen.js:37:20 in GooglePlacesAutocomplete.props.onPress
at node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js:270:25 in request.onreadystatechange
at node_modules/react-native/Libraries/Network/XMLHttpRequest.js:637:22 in setReadyState
at node_modules/react-native/Libraries/Network/XMLHttpRequest.js:395:24 in __didCompleteResponse
at node_modules/react-native/Libraries/vendor/emitter/EventEmitter.js:105:35 in emit

**Comment by jaredh159** [comment, marked as answer]
See this comment from this issue, it describes what i'm pretty certain is the problem and the solution:

https://github.com/jaredh159/tailwind-react-native-classnames/issues/184#issuecomment-1205151802

**Reply by EmpireNetworks** [reply]
Hello Jared, That was it. I was using a template literal in the form: style={tw`${!origin && 'opacity-20'}`}. Thank you so much for your great support and for this package!
